resources/scripts/openapi-icon-tool/.idea/misc.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wha does this file do?

Reply:
This files are auto added one. These are removed while removing `.idea` folder